intermediate/challenge-1/README.md

```diff
@@ -1,3 +1,21 @@
+# Build
+- mvn clean install
+
+# Docker
+- docker build -t audela/bus-challenge .
+- docker run -p 8080:8080 audela/bus-challenge
```

Comment:
I'd have liked to see a more elaborated  set up than just `docker run`

Reply:
I just added docker file to run it. I need to look/read more on docker documentation and need some play around to get a deep knowledge.